config/hooks/pre-commit

```diff
@@ -1,10 +1,22 @@
-#!/bin/sh -e
+#!/bin/bash -ex
```

Comment:
We should just do `-e`, not `-ex`, as we don't want the individual commands printed during each commit (at least I don't).

Reply:
Enacted in 72d981a09679c859b7b059d2ca2ae7bea75e4303.